.github/workflows/npmpublish.yml

```diff
@@ -7,6 +7,8 @@ on:
   push:
     branches:
       - master
+    tags:
```

Comment:
I think if we remove those 2 lines, it will trigger the script everytime, but because you added those if statements in those publish jobs, it will trigger them only on tags, right?

Reply:
I hope so!! I will try it now.

Reply:
it doesn't look like that did it.  I'm wondering if it will no longer try to run the `build` job since the PR is already open?